index.html

```diff
@@ -167,6 +167,7 @@ <h2>Contact me</h2>
                     <textarea name="message" id="message" cols="30" rows="10" placeholder="Write your message here"
                         maxlength="500" required></textarea>
                 </li>
+                <span id="emailError">Your email should be in lower case</span>
```

Comment:
- Your contact form is not being validated properly, according to the project requirements,  the form should not be submitted when there is a capital letter in the email, but I can still submit the form even when there is a capital letter. Please fix this 🙏 🙏 
- The email error in the form should only appear when the wrong email i.e an email containing an uppercase letter, is submitted, it should not be fixed on the HTML page. Please fix this 🙏